store/sqlstore/team_store.go

```diff
@@ -364,7 +364,7 @@ func (s SqlTeamStore) SearchAll(term string) ([]*model.Team, *model.AppError) {
 
 	term = sanitizeSearchTerm(term, "\\")
 
-	if _, err := s.GetReplica().Select(&teams, "SELECT * FROM Teams WHERE Name LIKE :Term OR DisplayName LIKE :Term", map[string]interface{}{"Term": term + "%"}); err != nil {
+	if _, err := s.GetReplica().Select(&teams, "SELECT * FROM Teams WHERE Name LIKE :Term OR DisplayName LIKE :Term", map[string]interface{}{"Term": "%" + term + "%"}); err != nil {
```

Comment:
I would build this in a small function. ie. wildcardSearchTerm(term)

Reply:
Good call. I have turned that into a function and added tests for both channel and teams.